sci-electronics/systemc/systemc-2.3.0-r1.ebuild

```diff
@@ -1,23 +1,26 @@
-# Copyright 1999-2013 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=4
 
 inherit eutils multilib toolchain-funcs autotools-utils
+MY_P="${P}a"
```

Comment:
Please do not use names that resemble normal version strings, i.e. `-r1` is better here.